CHANGELOG.md

```diff
@@ -2,8 +2,12 @@ Changelog for the gruf gem. This includes internal history before the gem was ma
 
 ### Pending release
 
+- Add ability to list, clear, insert before, and insert after to a server's interceptor
```

Comment:
I think it would be helpful to be able to remove an interceptor from the registry. Especially handy if one wants to remove one of the defaults without needing to clear other interceptors.